src/InstagramClone.Api/Controllers/UsersController.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Web.Http;
+using InstagramClone.Api.Database;
+using InstagramClone.Api.Entities;
+
+using Microsoft.EntityFrameworkCore;
+
+namespace InstagramClone.Api.Controllers
+{
+    
+    [RoutePrefix("api/users")]
+    public class UsersController : ApiController
+    {
+        private readonly InstagramCloneDbContext _context;
+        public UsersController(InstagramCloneDbContext context)
+        {
+            _context = context;
+        }
+           [HttpGet]
+        [Route("{id:Guid}")]
+        public IHttpActionResult GetUser(Guid id)
+        {
+            // user variable is used to store the value of a particuler user from the database
+            var user = _context.Users.Find(id);
```

Comment:
I expect a comment thaty rather explains what is happening don't the right-hand side of the `=`.  Please it should explain `_context.Users.Find(id);`

Reply:
???

Reply:
Ok.Done.